src/patternfly/components/Tabs/tabs.scss

```diff
@@ -531,6 +540,31 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
       content: revert;
     }
   }
+
+  &.pf-m-expanded {
+    --pf-c-tabs__item-toggle-icon--Color: var(--pf-global--Color--100);
+    --pf-c-tabs__item-toggle-icon--Rotate: var(--pf-c-tabs__item--m-expanded__toggle-icon--Rotate);
+  }
+
+  &:hover {
+    --pf-c-tabs__item-toggle-icon--Color: var(--pf-c-tabs__item--hover--Color);
+  }
+
+  &:active,
```

Comment:
Fun fact with the order of `:hover`, `:focus`, and `:active`! The order should always be hover -> focus -> active, since that's the order of events. If you style `:hover` after either `:focus` or `:active`, when using a mouse, you would never see the `:focus` or `:active` styles since both of those states require the mouse be on (hovering) the element. And if you style `:focus` after `:active`, you would never see the `:active` since with a mouse or keyboard, an `:active` element also has `:focus`.

So we just need to swap the order of focus and active here.